category_encoders/backward_difference.py

```diff
@@ -214,8 +214,7 @@ def transform(self, X, override_return_df=False):
         X = self.backward_difference_coding(X, mapping=self.mapping)
 
         if self.drop_invariant:
-            for col in self.drop_cols:
-                X.drop(col, 1, inplace=True)
+            X = X.drop(columns=self.drop_cols)
```

Comment:
Why'd you remove `inplace=True` here?

Reply:
I think the pandas dev team is not very happy with `inplace` and there's semi-regular attempts to get rid of it in general. I personally think inplace is not the most readable and transparent way to write pandas code and I try to avoid it.

There's an extensive discussion here: https://github.com/pandas-dev/pandas/issues/16529